### DISCUSSION TITLE
How to make a video call from telegram bot and connect it to another telegram user(known one) for video KYC?

### DISCUSSION THREAD

**Discussion by GurvinderkaurIT123** [python-telegram-bot/python-telegram-bot#2912] [Q&A]
Hello Dear Experts,

Greetings!!

First thing first, kudos for doing nice write up for "python-telegram-bot" in here, its a GREAT help for beginners like me. Appreciate it from bottom of my heart. Now coming on to my question :)

**_Question:_** I have built a basic Telegram chatbot using python-telegram-bot" libraries in Python3. So basically its a user verification bot which I am aiming to create, after some point of time I am stuck, now I have to build code where we need to have a functionality to do a video call between user(who is chatting with Telegram bot right now) AND with another known user(off course, eg: a support person from my firm; who will do video KYC for user).

I have tried searching in here and checked in link https://python-telegram-bot.readthedocs.io/en/stable/telegram.ext.html too but couldn't find any information related to making a video call. 

I apologies if I have missed something here, could experts please do guide me over here, will be grateful to you, cheers.

Thanks & Regards,
Guri

**Comment by Bibo-Joshi** [comment]
Hi. Glad to hear our lib is helpful :)

However, video calls is not something that bots can control - see also [this faq entry](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Frequently-Asked-Questions#can-you-add-feature-to-ptb-can-i-do-thing-with-my-bot).

**Reply by GurvinderkaurIT123** [reply]
Thank you sir for replying on same. Could you please do let me know if I try to add another Python code and try to integrate with Telegram will that work? Sorry I know its a noob question but want to know your views on same, Thank you.

**Reply by Bibo-Joshi** [reply]
It's surely possible to combine PTB with other frameworks, yes.

**Reply by GurvinderkaurIT123** [reply]
Thank you for your reply sir. Could you please do guide me how could I try to write it OR start it? I have seen Telegram API section honestly it was NOT at all clear :( 
Will be grateful to you if you could guide me how and where to start from in Telegram APIs interjection with Python3x.

Thanks & Regards,
Guri

**Reply by Poolitzer** [reply]
The go to python library for telegram (video) calls is https://github.com/pytgcalls/pytgcalls, in my opinion. Since it isn't possible to call a bot account, you need to have a user account for that.
